services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/skills_report/performance_breakdown/index.tsx

```diff
@@ -151,62 +152,38 @@ const PerformanceBreakdown = ({ practiceScope, setPracticeScope, classroom, open
-  //     <Route
-  //       element={
-  //         <History
-  //           sequencePractice={sequencePractice}
-  //           showEmptyState={noStudentHistoriesWithPreDiagnostic}
-  //           studentHistories={studentHistories}
-  //           teacherClassroomIds={teacherClassroomIds}
-  //           useELLDiagnostics={useELLDiagnostics}
-  //         />
-  //       }
-  //       path="by_class/history"
-  //     />
-  //     <Route
-  //       element={
-  //         <Recommendations
-  //           classroom={classroom}
-  //           previouslyAssignedDiagnosticsByLevel={previouslyAssignedDiagnosticsByLevel}
-  //           showEmptyState={noStudentHistoriesWithPreDiagnostic}
-  //           studentHistories={studentHistories}
-  //           useELLDiagnostics={useELLDiagnostics}
-  //         />
-  //       }
-  //       path="by_class/recommendations"
-  //     />
-  //     <Route
-  //       element={
-  //         <IndividualStudent
-  //           classroom={classroom}
-  //           previouslyAssignedDiagnosticsByLevel={previouslyAssignedDiagnosticsByLevel}
-  //           sequencePractice={sequencePractice}
-  //           showEmptyState={noStudentHistoriesWithPreDiagnostic}
-  //           studentHistory={selectedSkillCategory}
-  //           teacherClassroomIds={teacherClassroomIds}
-  //           timeframe={selectedTimeframe.label}
-  //           useELLDiagnostics={useELLDiagnostics}
-  //         />
-  //       }
-  //       path="by_student/:studentId"
-  //     />
-  //   </Routes>
-  // )
-
   const selectedSkillCategory = allSkillCategories.find(skillCategory => skillCategory.id === selectedSkillCategoryId)
+  const selectedSkillSubCategory = allSkillSubCategories.find(skillSubCategory => skillSubCategory.id === selectedSkillSubCategoryId)
+
+  const routes = (
+    <Routes>
+      <Route
+        element={
+          <ByStudent
+            classroom={classroom}
+            momentumScoreGoal={momentumScoreGoal}
+            practiceScope={practiceScope}
+            selectedSkillCategory={selectedSkillCategory}
+            selectedSkillSubCategory={selectedSkillSubCategory}
+          />
+        }
+        path="by_student"
+      />
+    </Routes>
+  )
+
+  if (!classroom || !selectedSkillCategory) { return <Spinner /> }
 
   return (
-    <main>
+    <main className="performance-breakdown">
       <header>
         <h1>
           Performance Breakdown
           {byStudent ? <span>/ By Student</span> : <span>/ By Skill</span>}
           {selectedSkillCategory ? <span>/ {selectedSkillCategory.name}</span> : null}
         </h1>
         <div className="links">
-          <a className="quill-button outlined small grey focus-on-light" href='' rel="noopener noreferrer" target="_blank">
+          <a className="quill-button outlined small grey focus-on-light" href='https://support.quill.org/en/articles/12062553-how-do-i-read-the-skills-report-quill-premium' rel="noopener noreferrer" target="_blank">
```

Comment:
We're already importing constants from `./shared`; this could be an imported constant as well